### ISSUE TITLE
Remember patch load order

### ISSUE THREAD

**Issue by Camburgaler** [AntonioDePau/KHPCPatchManager#52]
# CURRENT BEHAVIOR
Every time there is any change in what patches to use via the "Select patch" button, one must reorder the patches, even if most of the patches are the same.

# PROBLEM STATEMENT
This is inconvenient. I would like to be able to slap one additional patch into my patch list without having to reorder the entire list.

# TARGET BEHAVIOR
Remember the load order of all previously loaded patches, or even just the patches that were loaded most recently. A simple implementation that comes to mind is simply storing this data in a .txt or .csv file.

`loadOrder.csv` e.g.:
```
0,Patch_A.kh2pcpatch
1,Patch_B.kh2pcpatch
2,Patch_C.kh2pcpatch
...
A0,Patch_D.kh2pcpatch
A1,Patch_E.kh2pcpatch
A2,Patch_F.kh2pcpatch
...
```

**Comment by AntonioDePau** [comment]
Good idea. Although I might be lazy enough to allow .txt to be a selectable format and if the user selects a .txt file, parse its lines, with each line being the path to a .*pcpatch file (ignoring the ones that don't end in .*pcpatch) and use those as the input files.

Will think about it, thanks!
